listenbrainz/webserver/views/feedback_api.py

```diff
@@ -225,13 +225,21 @@ def get_feedback_for_recordings_for_user(user_name):
     :statuscode 200: Yay, you have data!
     :resheader Content-Type: *application/json*
     """
-
-    msids_unparsed = request.args.get("recording_msids")
-    if msids_unparsed is None:
-        msids_unparsed = request.args.get("recordings")
-    mbids_unparsed = request.args.get("recording_mbids")
+    msids_unparsed = None
+    mbids_unparsed = None
 
     recording_msids, recording_mbids = [], []
+    if request.method == "GET":
+        msids_unparsed = request.args.get("recording_msids")
+        if msids_unparsed is None:
+            msids_unparsed = request.args.get("recordings")
+        mbids_unparsed = request.args.get("recording_mbids")
+    elif request.method == "POST":
+        msids_unparsed = request.json.get(recording_msids)
+        if msids_unparsed is None:
```

Comment:
You can avoid this fallback for the POST case because it is a new addition and no one is using it.

Reply:
No, the GET is under use. I just meant that you can remove the check for `recordings` field from POST. That check is only needed in GET.